api/cloudcontroller/ccv2/errors_test.go

```diff
@@ -16,6 +16,8 @@ var _ = Describe("Error Wrapper", func() {
 	var (
 		serverResponse     string
 		serverResponseCode int
+		requestMethod      string
```

Comment:
It seems like you're setting this up so that the tests can serve as more accurate documentation and simulate the real world more closely. However, it feels odd to have this "real-world-ness" just for the service broker endpoint, but all other errors in this file continue to just call `GET /v2/apps` with `client.GetApplications()`. If we want to clarify that the service broker error should generally only occur when you hit the endpoint for creating service brokers, then why not do the same for the space-creation failures, buildpack not found errors, etc.? Just feels weird that only service broker creation follows this style and the rest just rely on the fact that the error wrapper is super generic and doesn't care what request was made.

Reply:
Thanks. We've moved the call to `client.GetApplications()` to a JustBeforeEach, and added a comment to explain that we're stubbing the call so the actual endpoint really matter.